packages/outputarea/src/model.ts

```diff
@@ -526,6 +526,9 @@ namespace Private {
     if (text === undefined) {
       text = '';
     }
+    if (!(newText.includes('\b') || newText.includes('\r'))) {
```

Comment:
Should it also consider `'\n'`? There is some special-casing for it in the logic?

Reply:
No, `\n` is not a special character unless there is `\b` or `\r` in the text.